packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -391,16 +440,26 @@ export class IamPolicyChecksWebview extends VueWebview {
             }
             case 'CloudFormation': {
                 if (isCloudFormationTemplate(document)) {
-                    const cfnCommand =
-                        cfnParameterPath === ''
-                            ? `cfn-policy-validator check-access-not-granted --template-path ${document} --region ${this.region} --actions ${actions}`
-                            : `cfn-policy-validator check-access-not-granted --template-path ${document} --region ${this.region} --actions ${actions} --template-copnfiguration-file ${cfnParameterPath}`
```

Comment:
`--template-copnfiguration-file` based on other uses in this file, it seems this is a one off typo

Reply:
Thanks for catching this!